evap/staff/templates/staff_index.html

```diff
@@ -23,11 +23,11 @@
         </div>
         <div class="col-md-3">
             <h3>{% trans "Questionnaires" %}</h3>
+            <a href="{% url "evap.staff.views.questionnaire_index" %}">{% trans "All questionnaires" %}</a>
```

Comment:
@janno42, if you could check this out and see whether you can pretty it up, that would be cool.